homeassistant/components/upnp.py

```diff
@@ -52,7 +57,7 @@
 
 
 # pylint: disable=import-error, no-member, broad-except, c-extension-no-member
```

Comment:
Is this still needed?

Reply:
Probably not...

I'm still working on the tests (Mocking async methods is not trivial), will check pylint at the end.

Reply:
no longer needed